### PR TITLE
Update README with apiserver setup docs

### PR DESCRIPTION
**Pull request by amitmalav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +38,19 @@ git clone https://github.com/sap/service-fabrik-boshrelease
 cd service-fabrik-boshrelease
 ```
 
+### Deploying ApiServer
+Apiserver is a prerequisite for running Service-Fabrik-Broker. It can be deployed on boshlite using provided manifest [file](https://github.com/cloudfoundry-incubator/service-fabrik-boshrelease/blob/master/templates/apiserver.yml)
```
Comment (subhankarc) [inline README.md]: Apiserver uses ETCD as backend. So, ETCD is also a prerequisite.

`Apiserver along with ETCD` can be deployed...............

File: README.md [hunk 2]
```diff
@@ -29,6 +38,19 @@ git clone https://github.com/sap/service-fabrik-boshrelease
...
+Prerequisites:
+apiserver has dependency on cfcr-etcd release, which have to be first uploaded on bosh before deploying apiserver.
```
Comment (subhankarc) [inline README.md]: Direct upload command can be given without downloading the source

File: README.md [hunk 3]
```diff
@@ -5,6 +5,15 @@ A bosh release to deploy Service Fabrik which provisions service instances as Do
 ## 3rd Party
 This BOSH release is based on/forked from and therefore includes sources from the following 3rd party BOSH release: https://github.com/cloudfoundry-community/docker-boshrelease, [Apache License Version 2.0](https://github.com/cloudfoundry-community/docker-boshrelease/blob/master/LICENSE).
 
+# Table of Contents
+1. [Local Development Setup](#local-development-setup)
+2. [Cloning the Repository](#cloning-the-repository)
+3. [Deploying ApiServer](#deploying-apiServer)
```
Comment (ketakig12) [inline README.md]: Looks like this link is broken. Please check once.

File: README.md (hunk 1 above)
Comment (ketakig12) [reply README.md]: I needed to update bosh-cloudconfig with below address before deploying apiserver. Is this step somewhere else?
```
static:
  - 10.244.0.2 - 10.244.14.254

bosh -e boshlite update-cloud-config cloud-config.yml
```

Comment (amitmalav) [reply README.md]: cloud-config present in boshrelease is outdated. Raised PR #116  with working cc

File: README.md (hunk 3 above)
Comment (amitmalav) [reply README.md]: Fixed.

**Pull request merged**